Backend/routes/customerOrderRoutes.js

```
@@ -21,109 +25,119 @@ router.get("/", async (req, res) => {
-    if (retailerOrder) {
-      const orderTotal = parseFloat(order.total.replace("$", ""));
-      retailerOrder.total = `$${(
-        parseFloat(retailerOrder.total.replace("$", "")) - orderTotal
-      ).toFixed(2)}`;
-      if (retailerOrder.total === "$0.00") {
-        await RetailerOrder.deleteOne({ _id: retailerOrder._id });
+      let retailerOrder = await RetailerOrder.findOne({ shopId });
+      if (retailerOrder) {
+        retailerOrder.items = retailerOrder.items.concat(items);
+        retailerOrder.total = `$${(
+          parseFloat(retailerOrder.total.replace("$", "")) + totalAmount
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)